lib/passport-azure-ad/metadata.js

```diff
@@ -102,41 +121,128 @@ Metadata.prototype.updateWsfedMetadata = function(doc, next) {
   }
 };
 
+Metadata.prototype.updateOidcMetadata = function(doc, next) {
+  log.info('Request to update the Open ID Connect Metadata');
+  try {
+    this.oidc = {};
+    this.oidc.issuer = doc['issuer'];
+    this.oidc.algorithms = doc['id_token_signing_alg_values_supported'];
```

Comment:
I use it in line 161-167 of `oidcstrategy.js`. It's in the options array passed to the jwt.verify()

```
if (!options.issuer) {
      options.issuer = this.metadata.oidc.issuer;
    }
    options.algorithms = this.metadata.oidc.algorithms;

    jwt.verify(token, PEMkey, options, function(err, token) {
```